gm4_bookshelf_nametags/data/gm4_bookshelf_nametags/functions/evaluate/get_pos.mcfunction

```diff
@@ -0,0 +1,9 @@
+# spawn marker to get player position
+# @s = marker
+# at unspecified
+# run from evaluate/motion
+
+tp @s @p[tag=gm4_bookshelf_nametags_target]
```

Comment:
Whilst it doesn't make a difference anymore, I'd prefer to use `@a` instead of `@p` for readability. (not critical)
```suggestion
tp @s @a[tag=gm4_bookshelf_nametags_target,limit=1,sort=nearest]
```

Reply:
Going to disagree with this one, I find the simplicity of not having an extra `,limit=1,sort=nearest` to make it more readable